src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1298,13 +1298,13 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_Microsoft_PowerShell_Co
                         .AddItemScriptBlock(@"
                                   $result = $_.Content
                                   $result = $result.Substring(0, [Math]::Min($result.Length, 200) )
-                                  if($result.Length -eq 200) { $result += ""..."" }
+                                  if($result.Length -eq 200) { $result += ""`u{2026}"" }
```

Comment:
Just an FYI.  I've seen some places where unicode doesn't render with the change to using the null replacement character, but they are few.

Reply:
Discussed offline, we are ok with this change and can revisit pending customer feedback as the major distros support rendering.